pkg/volume/csi/nodeinfomanager/nodeinfomanager_test.go

```diff
@@ -770,6 +845,20 @@ func TestInstallCSIDriverExistingAnnotation(t *testing.T) {
 	}
 }
 
+func getClientSet(existingNode *v1.Node, existingCSINode *storage.CSINode) *fake.Clientset {
+	var client *fake.Clientset
+	if existingCSINode != nil && existingNode != nil {
+		client = fake.NewSimpleClientset(existingNode, existingCSINode)
```

Comment:
Does NewSimpleClientset not handle nil values passed into it?

Reply:
No, but I can simplify this method.